base_name_search_improved/README.rst

```diff
@@ -82,9 +82,9 @@ the top right search box, is not affected.
 Additional search fields can be configured at Settings > Technical >
 Database > Models, using the "Name Search Fields" field.
 
-|image1|
+|image3|
 
-.. |image1| image:: https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image1.png
+.. |image3| image:: https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image1.png
```

Comment:
This almost works, but the README.rst is generated using `oca-gen-addon-readme` from https://github.com/OCA/maintainer-tools. You can install this tool locally in your python env and run it using `ca-gen-addon-readme --repo-name server-tools --branch 17.- --addon-dir .` if you change into the module directory on your setup.

You need to make the change in the files in `readme`. 

```
--- a/base_name_search_improved/readme/CONFIGURE.md
+++ b/base_name_search_improved/readme/CONFIGURE.md
@@ -5,4 +5,4 @@ the top right search box, is not affected.
 Additional search fields can be configured at Settings \> Technical \>
 Database \> Models, using the "Name Search Fields" field.
 
-![](https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image1.png)
+![image1](https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image1.png)
diff --git a/base_name_search_improved/readme/DESCRIPTION.md b/base_name_search_improved/readme/DESCRIPTION.md
index 88b7ee5448..e117da5604 100644
--- a/base_name_search_improved/readme/DESCRIPTION.md
+++ b/base_name_search_improved/readme/DESCRIPTION.md
@@ -10,13 +10,13 @@ relaxed search also looks up for records containing all the words, so
 "John M. Brown" would be a match. It also tolerates words in a different
 order, so searching for "brown john" also works.
 
-![](https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image0.png)
+![image0](https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image0.png)
 
 Additionally, an Administrator can configure other fields to also lookup
 into. For example, Customers could be additionally searched by City or
 Phone number.
 
-![](https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image2.png)
+![image2](https://raw.githubusercontent.com/OCA/server-tools/11.0/base_name_search_improved/images/image2.png)
 
 How it works:
 
```